# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/RemoteImageHolder.h
# Commit: 08047518429b
# Full Hash: 08047518429b15fccb5616a5ad18fc19cefd65b1
# Author: Brad Werth <bwerth@mozilla.com>
# Date: 2022-09-12 21:40:55
# Description:
#   Bug 1790436: Provide a default value for  RemoteImageHolder::mColorDepth. r=alwu
#   
#   The default constructor for RemoteImageHolder assigns default values to
#   all of its members. gfx::ColorDepth does not define a default value. This
#   change makes it so RemoteImageHolder provides a default value.
# ==============================================================================

diff -r 880f0718c9ba -r 08047518429b dom/media/ipc/RemoteImageHolder.h
--- a/dom/media/ipc/RemoteImageHolder.h	Mon Sep 12 17:24:48 2022 +0000
+++ b/dom/media/ipc/RemoteImageHolder.h	Mon Sep 12 17:25:28 2022 +0000
@@ -48,7 +48,7 @@
   // We need a default for the default constructor, never used in practice.
   layers::VideoBridgeSource mSource = layers::VideoBridgeSource::GpuProcess;
   gfx::IntSize mSize;
-  gfx::ColorDepth mColorDepth;
+  gfx::ColorDepth mColorDepth = gfx::ColorDepth::COLOR_8;
   Maybe<layers::SurfaceDescriptor> mSD;
   RefPtr<layers::IGPUVideoSurfaceManager> mManager;
 };
